firmware/spade/src/pc/CMakeLists.txt

```diff
@@ -24,3 +26,5 @@ set(CMAKE_CXX_CLANG_TIDY
   -format-style='file';
   -header-filter=${CMAKE_CURRENT_SOURCE_DIR};
 )
+
+add_compile_definitions(IS_RPI=0)
```

Comment:
The pc makefile shouldn't need to know *anything* about rpi, the fact that it does is evidence of something that's probably wrong...